packages/core/src/metrics/MetricsService.ts

```diff
@@ -47,4 +42,20 @@ export abstract class MetricsService {
         return this.publisher.publish(payload);
     }
 
+    /**
+     * Generates or gets mobile client id
+     */
+    public getClientId(): string {
+        let clientId = this.getStoredClientId();
+
+        if (!clientId) {
+            clientId = "TODO: generate a uuid";
```

Comment:
@wtrocki would we use here a `UUIDGenerator` class like in https://github.com/feedhenry/fh-sync-js/blob/master/src/clientIdProvider.js#L17 ?

Reply:
Why not use it then ?